dotCMS/src/main/java/com/dotmarketing/cms/urlmap/URLMapAPIImpl.java

```diff
@@ -246,7 +247,8 @@ private Contentlet getContentlet(
             checkContentPermission(context, contentlet);
         }
 
-        return contentlet;
+       return new DotTransformerBuilder().
+               defaultOptions().content(contentlet).build().hydrate().get(0);
```

Comment:
maybe we should add more asserts in this test

https://github.com/dotCMS/core/blob/5f9ad4da39ca791f7866d8844d9bb59b0bf35617/dotCMS/src/integration-test/java/com/dotmarketing/cms/urlmap/URLMapAPIImplTest.java#L229

Reply:
Done